src/Functions/FunctionsStringArray.h

```diff
@@ -98,6 +98,12 @@ class AlphaTokensImpl
         return 0;
     }
 
+    /// Returns the position of the possible max_substrings argument. std::nullopt means max_substrings argument is disabled in current function.
```

Comment:
No need to change anything, just as a hint: Having the positions (l. 96, l. 102) as static constants similar to name (l. 70) would have been a little bit more elegant.

Reply:
Thanks. I agree with you. Maybe there is performance benefit after changing. Since currently we get string positions from static functions in FunctionTokens, let's keep max_substrings positions the same.